reference-implementation/to-upstream-wpts/writable-streams/bad-underlying-sinks.js

```diff
@@ -47,6 +77,48 @@ promise_test(t => {
 }, 'close: returning a rejected promise should cause writer close() and ready to reject');
 
 promise_test(t => {
+  const ws = new WritableStream({
+    get close() {
+      throw error1;
+    }
+  });
+
+  const writer = ws.getWriter();
+
+  return promise_rejects(t, error1, writer.close(), 'close should reject with the thrown error');
+}, 'close: throwing getter should cause writer close() to reject');
+
+promise_test(t => {
+  const ws = new WritableStream({
+    get write() {
+      throw error1;
+    }
+  });
+
+  const writer = ws.getWriter();
+
+  return promise_rejects(t, error1, writer.write('a'), 'write should reject with the thrown error')
+  .then(() => {
+    return promise_rejects(t, error1, writer.closed, 'closed should reject with the thrown error');
```

Comment:
That's weird. I intended to write {} but it came out as ().

What I was trying to say is you could just do `() => promise_rejects(t, error1, writer.closed, 'closed should reject with the thrown error')`.